static/game/pausescreen.js

```diff
@@ -33,7 +33,7 @@ export class PauseScreen extends Menu {
     addToGui("stats", stats);
     this.childMenus = [codex, stats];
 
-    this.items = [
+    this.setItems([
```

Comment:
This setter is required because I need to recalculate the list visual height when items are changed.